Content.Server/Medical/MedicalScannerSystem.cs

```diff
@@ -78,29 +78,27 @@ private void OnRelayMovement(EntityUid uid, MedicalScannerComponent scannerCompo
 
         private void AddInsertOtherVerb(EntityUid uid, MedicalScannerComponent component, GetVerbsEvent<InteractionVerb> args)
         {
-            if (args.Using == null ||
-                !args.CanAccess ||
-                !args.CanInteract ||
-                IsOccupied(component) ||
-                !CanScannerInsert(uid, args.Using.Value, component))
+            if (args.Using == null
+                || !args.CanAccess
+                || !args.CanInteract
+                || IsOccupied(component)
+                || !CanScannerInsert(uid, args.Using.Value, component)
+                || !TryComp<MetaDataComponent>(args.Using.Value, out var metadata))
                 return;
 
-            var name = "Unknown";
-            if (TryComp<MetaDataComponent>(args.Using.Value, out var metadata))
-                name = metadata.EntityName;
-
             InteractionVerb verb = new()
             {
                 Act = () => InsertBody(uid, args.Target, component),
                 Category = VerbCategory.Insert,
-                Text = name
+                Text = metadata.EntityName
             };
             args.Verbs.Add(verb);
         }
 
         private void AddAlternativeVerbs(EntityUid uid, MedicalScannerComponent component, GetVerbsEvent<AlternativeVerb> args)
         {
-            if (!args.CanAccess || !args.CanInteract)
+            if (!args.CanAccess
+                || !args.CanInteract)
```

Comment:
```suggestion
            if (!args.CanAccess || !args.CanInteract)
```
Doesn't seem like any reason to break this one up.